config/Dockerfiles/grafana/Dockerfile

```diff
@@ -1,22 +1,19 @@
 FROM centos:7
-ENV GRAFANA_VERSION=5.0.0
 
-LABEL name="Grafana" \
-      version=$GRAFANA_VERSION
+ENV GRAFANA_INSTALL grafana-5.0.0-1.x86_64.rpm
```

Comment:
you could do one ENV then just keep pasting them *i think* but just a nit pick

Reply:
Looks like you can. Testing it now.